<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generateOrders</class>
 <widget class="QDialog" name="generateOrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>439</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Generate Court Orders</string>
  </property>
  <widget class="QLabel" name="label_generateTitle">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>65</y>
     <width>266</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GENERATE COURT ORDERS</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>75</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>116</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE AND PRINT</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_caseNumber">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>90</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Case Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>120</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Master Order</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_masterOrder">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>117</y>
     <width>116</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DV110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DV130</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DV140</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CH130</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EA110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EA130</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other Orders</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame_attachments">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>155</y>
     <width>331</width>
     <height>136</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="checkBox_generateDV140">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>146</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DV140 CC - CV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateDV145">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DV145 Travel Restrict</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateDV150">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DV150 Supervision</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateFL340">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>5</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FL340</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateFL341">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FL341</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_otherForm">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>55</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_otherAttach1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_otherAttach2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>105</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>80</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Other Attachment 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Other Attachment 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateFL342">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>5</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FL342</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_generateFL343">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FL343</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>155</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Attachments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Other Orders</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DV730">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>300</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DV730 - Renew Orders</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DV115">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>325</y>
     <width>236</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DV115 - Request for Continuance</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DV116">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>350</y>
     <width>236</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DV116 - Reissuance Orders</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_eSign">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>eSIGN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>generateOrders</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>generateOrders</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
